pkg/printers/internalversion/printers.go

```diff
@@ -1764,8 +1764,7 @@ func printNode(obj *api.Node, options printers.GenerateOptions) ([]metav1.TableR
 
 	conditionMap := make(map[api.NodeConditionType]*api.NodeCondition)
 	NodeAllConditions := []api.NodeConditionType{api.NodeReady}
-	for i := range obj.Status.Conditions {
-		cond := obj.Status.Conditions[i]
+	for _, cond := range obj.Status.Conditions {
```

Comment:
can this change be susceptible of https://www.evanjones.ca/go-gotcha-loop-variables.html ? old code seems safe in this regard

Reply:
https://github.com/golang/go/issues/60078
It seems that it is. It is more appropriate to close this one.
/close